IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" PrimaryDataType="IfcLabel" />
```

Comment:
Description of the property would be very helpful. Code implies a consideration to change the datatype to `IfcIdentifier`.

Reply:
Ik_code is not realy an identifier. It correspond to a numeric classification for the degree of protection. Label ok ?
Description come from IEC.

Reply:
Ok from meeting review